wiki/Guides/de.md

```diff
@@ -0,0 +1,54 @@
+<!-- TODO: change link as soon as German article is available (line 12) -->
+
+# Tutorials
+
+Unten findet sich eine Liste an Tutorials, die von Mitgliedern der osu!-Community erstellt wurden. Die meisten Tutorials entstanden in den osu!-Foren und wurden in das osu!-Wiki integriert.
+
+## Beatmapping
+
+*Hauptseite: [Beatmapping](/wiki/Beatmapping)*
+
+- [Ratgeber für das Bearbeiten von Audio](Audio_Editing)
+- [Den Titeltext ändern](/wiki/Beatmap/Title_Text#changing-title-text)
+- [Dateien komprimieren](Compressing_files)
+- [Eigene Bibliothek für Hitsounds](Custom_Hitsound_Library)
+- [Benutzerdefinierte Überschreibung von Audiosamples](Custom_Sample_Overrides)
+- [Wie man den Künstler oder Titel einer Map ändert](Changing_the_Artist_or_Title)
+- [Wie man Probleme mit dem BSS löst](BSS_Issues)
+- [Wie man eine Beatmap-Kollaboration startet](Collab_Information)
+- [Wie man ein Beatmap-Projekt anfängt](Starting_a_Beatmap_Project)
+- [Wie man Songs timt](How_to_Time_Songs)
```

Comment:
i thought `timt` looked really wrong but i checked the [duden website](https://www.duden.de/konjugation/timen) and its actually correct... lol weird

Reply:
Yeah, I looked that up and my mind was as blown as yours xD